packages/forms-web-app/__tests__/unit/views/macros/__snapshots__/cookie-banner-accepted.njk.test.js.snap

```diff
@@ -12,8 +12,7 @@ exports[`views/macros/cookie-banner-accepted should allow setting a custom cooki
     
     
 
-    <div class=\\"govuk-cookie-banner__message govuk-width-container\\"
-    >
+    <div class=\\"govuk-cookie-banner__message govuk-width-container\\">
```

Comment:
For some reason, doing a node upgrade meant the snapshots are formatted differently, although the text seems to be the same